Learn-JSON-fetch-promises-async-await-by-building-an-fCC-forum-leaderboard/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>
+      Learn about the Fetch API, promises and Async await by building an
+      freeCodeCamp forum leader board
+    </title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="https://forum.freecodecamp.org/"
+              target="_blank"
+              rel="noreferrer noopener"
+            >
+              <img
+                class="fcc-logo"
+                src="./fcc_primary_large.png"
```

Comment:
```suggestion
                src="https://cdn.freecodecamp.org/platform/universal/fcc_primary.svg"
```

Rather than having to host another copy of this, we could just fetch it from our CDN

Reply:
that makes sense